prometheus/README.md

```diff
@@ -1,55 +1,156 @@
-# Setting up Prometheus
+# Setting up Prometheus for pgmonitor
 
-## Installation (RHEL/CENTOS 7)
+Prometheus can be set up on any Linux-based system, but the instructions below use RHEL/CentOS 7.
 
-There are RPM packages available to [Crunchy Data](https://www.crunchydata.com) customers through the Crunchy Data [access portal](https://access.crunchydata.com/).  Installing these RPMs will take care of all of the steps described in this Installation section. If you install via the RPM, you can continue reading at the [Setup](#setup-rhelcentos-7) section.
+- [Installation](#installation)
+- [Setup](#setup)
+   - [RHEL / CentOS 7](#setup-on-rhelcentos-7)
 
-Packages available: prometheus2, alertmanager, pgmonitor-prometheus-extras, pgmonitor-alertmanager-extras
+## Installation
 
-For non-package installations, Prometheus & Alertmanager can be downloaded from the developer website (https://prometheus.io/download). The minimum expected versions are Prometheus is 2.0 and Alertmanager 0.12.0. The files contained in this repository are assumed to be installed in the following locations with the following names:
+### Installation on RHEL/CentOS 7
+
+#### With RPM Packages
+
+There are RPM packages available to [Crunchy Data](https://www.crunchydata.com) customers through the [Crunchy Customer Portal](https://access.crunchydata.com/).
+
+If you install the below available packages with RPM, you can continue reading at the [Setup](#setup) section.
+
+##### Available Packages
+
+| Package Name                  |
+|-------------------------------|
+| alertmanager                  |
+| pgmonitor-alertmanager-extras |
+| pgmonitor-prometheus-extras   |
+| prometheus2                   |
+
+#### Without Packages
+
+For installations without using packages, you can download [Prometheus](https://prometheus.io/) and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) from [https://prometheus.io/download](https://prometheus.io/download).
+
+##### Minimum Versions
+
+|              | Minimum Version |
+|--------------|-----------------|
+| Prometheus   | 2.0             |
+| Alertmanager | 0.12.0          |
+
```

Comment:
I would prefer not to list specific version names in the readme like this. It's very easy to forget to update it. Can mention that our build assumes Prometheus 2.x and should always use the latest minor versions of both Prometheus and the Alertmanager.

Reply:
English doesn't seem coherent here. I would just say "Prometheus 2.x" and not mention any version of Alertmanager at all.

Reply:
@keithf4 Updated.